src/Container.js

```diff
@@ -107,7 +108,7 @@ class Container {
   getTaggedService(tag) {
     return Array
       .from(this.services.values())
-      .filter(definition => definition.tag === tag)
+      .filter(definition => tag in definition.tags)
```

Comment:
The `in` notation only work for object, you need `includes` here:
`definition.tags.includes(tag)`